### ISSUE TITLE
Is there any planning on Cpp support

### ISSUE THREAD

**Issue by SSoulaimane** [jacob-carlborg/dstep#131]
Is there any plan to support Cpp.

**Comment by jacob-carlborg** [comment]
Yes, eventually. But that's not something I'm going to work on in the near future. Any help is appreciated.

**Comment by Swoorup** [comment]
How difficult is it right now? @jacob-carlborg ?

I wonder if majority of parsing stuff that can done via clang? Sorry have no idea. I thought to scope this out before I try to crack it

**Comment by jacob-carlborg** [comment]
All the source code analyzing is done by libclang (lexing, parsing, semantic analyze). Then DStep "only" needs to walk the AST provided by libclang and generate D code from it.
